### ISSUE TITLE
Suggestion: Compliments reaction 

### ISSUE THREAD

**Issue by DerperCrack00** [Monika-After-Story/MonikaModDev#8254]
#8243 So, when you compliment Monika for the first time, you get a different dialogue with different options, and that only happens the first time you compliment her about a specific thing, and I'd like it if there was a small chance of getting those options again, even after having complimented her.

**Comment by WiiBii2** [comment]
Yeah I'm working on something like this where I extend different responses to things you say to Monika. I've been putting it off for a while, but I'll get back to working on it.

**Comment by shinolife63** [comment]
@WiiBii2 If you need any help or want to bounce ideas off someone, lmk; I've been considering the same thing and wouldn't mind comparing notes.

**Comment by ThePotatoGuy** [comment]
[leaving open as in progress]
